nitrite.go

```diff
@@ -460,3 +460,23 @@ func checkECDSASignature(
 
 	return ecdsa.Verify(publicKey, hashSigStruct, r, s)
 }
+
+func (d *Document) FromCosePayloadBytes(bytes []byte) error {
+	cose := CosePayload{}
+	err := cbor.Unmarshal(bytes, &cose)
+	if nil != err {
+		return fmt.Errorf("unmarshaling CosePayload: %w", err)
+	}
+
+	err = cbor.Unmarshal(cose.Payload, d)
+	if nil != err {
+		return fmt.Errorf("unmarshaling Document: %w", err)
+	}
+	return nil
+}
+
+func (d *Document) CreatedAt() time.Time {
+	// Pg. 64 of https://docs.aws.amazon.com/pdfs/enclaves/latest/user/enclaves-user.pdf
+	// describes Timestamp as "UTC time when document was created, in milliseconds"
+	return time.UnixMilli(int64(d.Timestamp))
+}
```

Comment:
Per our convo over video chat, this code was taken from the [nitriding PR](https://github.com/blocky/nitriding/pull/7/files#diff-8cc90ea4d00e2071d0e51060f99c95b2b0599411d16f379a61a98e6523d3996d) and moved here into nitrite. Essentially, we have a helper function for unmarshaling an attestation document from the CosePayload byte array and one for extracting the created at time in milliseconds.